CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_ad_reporting v1.2.0
```

Comment:
In this changelog entry I would highlight that if existing customers are not using Amazon Ads, then they can simply add the following variable to their dbt_project.yml
```yml
vars:
  ad_reporting__amazon_ads_enabled: False ## True by default
```

Reply:
Done